pages/create-collection/index.js

```diff
@@ -0,0 +1,96 @@
+import CreateNav from "@/components/CreateNav";
+import { useState } from "react";
+import styled from "styled-components";
+import Button from "@/components/Button";
+
+const StyledHeadline = styled.h1`
+  display: flex;
+  justify-content: center;
+`;
+
+const StyledFormContainer = styled.div`
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  width: 100vw;
+  padding: 3em;
+`;
+
+const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  width: 250px;
+  gap: 10px;
+`;
+
+const StyledColors = styled.div`
+  width: 24px;
+  height: 24px;
+  border-radius: 50px;
+  cursor: pointer;
+`;
```

Comment:
I suggest to use `buttons` here instead of `div` elements for better accessibility.